scripts/constants.js

```diff
@@ -3,8 +3,11 @@ module.exports = {
   MAX_LOCATORS: 10,
   APPROVAL_AMOUNT: '90071992547409910000000000',
   INDEX_HEAD: '0xFFfFfFffFFfffFFfFFfFFFFFffFFFffffFfFFFfF',
-  NULL_ADDRESS: '0x0000000000000000000000000000000000000000',
   DEFAULT_CONFIRMATIONS: 2,
+  rinkebyTokens: {
+    DAI: '0x5592ec0cfb4dbc12d3ab100b257153436a1f0fea',
```

Comment:
do these need to be checksummed?

Reply:
They do not need to be checksummed. Let me know if you have something in mind

Reply:
INDEX_HEAD: '0xFFfFfFffFFfffFFfFFfFFFFFffFFFffffFfFFFfF' is checksummed which was why i asked.

Reply:
if not required, all good!